WordPressKit/WordPressKit/RemoteBlogJetpackMonitorSettings.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/// This struct encapsulates the *remote* Jetpack monitor settings available for a Blog entity
+///
+public struct RemoteBlogJetpackMonitorSettings {
+
+    /// Indicates whether the Jetpack site's monitor notifications should be sent by email
+    ///
+    public let monitorEmailNotifications: Bool
+
+    /// Indicates whether the Jetpack site's monitor notifications should be sent by push notifications
+    ///
+    public let monitorPushNotifications: Bool
+
+    public init(monitorEmailNotifications: Bool,
```

Comment:
Do we need to declare the initializer, or can we have Swift automatically do that? I see there's not one for `RemoteBlogJetpackSettings`

Reply:
If I don't declare it is marked as internal by default and I can't use it on `BlogJetpackSettingsService`. I don't know if there's a way around that,  do you?

Reply:
Ah I saw `RemoteBlogJetpackSettings` didn't have the initializer and it confused me, but that one is only initialized from the remote code. 👍